OpenSim/Simulation/Model/BushingForce.h

```diff
@@ -80,15 +75,28 @@ OpenSim_DECLARE_CONCRETE_OBJECT(BushingForce, Force);
 //==============================================================================
 // PUBLIC METHODS
 //==============================================================================
-    /** Default constructor leaves bodies unspecified, sets the bushing frames
+    /** Default constructor leaves frames unspecified, sets the bushing frames
     to be at their body origins, and sets all bushing parameters to zero. **/
     BushingForce();
-    /** This constructor provides everything needed to create a real 
-    bushing. See property declarations for more information. **/
-    BushingForce(const std::string& body1Name, 
+
+    /** Construct the BushingForce given the offset frames that it tries to
+    align according to the physical properties of the bushing.
+    See property declarations for more information. **/
+    BushingForce(const PhysicalOffsetFrame& frame1,
```

Comment:
I don't understand why all these constructors are needed and why the BushingForce doesn't use `PhysicalFrame`. Isn't a `PhysicalOffsetFrame` a kind of `PhysicalFrame`? I would think the only need constructor would be
```c++
BushingForce(
    const PhysicalFrame& frame1,
    const PhysicalFrame& frame2,
    const SimTK::Vec3& transStiffness,
    const SimTK::Vec3& rotStiffness,
    const SimTK::Vec3& transDamping,
    const SimTK::Vec3& rotDamping);
```
Why isn't that good enough?